.github/workflows/publish-HOWTOs.yml

```diff
@@ -87,6 +87,13 @@ jobs:
           name: workflow-howtos-aws
           path: public-aws/
 
+      - name: Get Archive zip from GH URL
+        uses: suisei-cn/actions-download-file@v1
+        id: downloadfile
+        with:
+          url: https://github.com/built-on-openfin/workspace-starter/archive/refs/heads/workspace/${{ github.ref_name }}.zip
```

Comment:
Hi Jeff.

I believe github.ref_name would be workspace/v8.0.0 and this would have workspace twice. Shouldn't it be:  https://github.com/built-on-openfin/workspace-starter/archive/refs/heads/${{ github.ref_name }}.zip

And does this only execute if the branch name starts with workspace/ (so we only publish release branches and not dev branches) ?

Reply:
yep you're right that would double the path.. corrected